cmd/root.go

```diff
@@ -35,7 +35,7 @@ var (
 
 const (
 	// ShortTimeFormat is a short format for printing timestamps
-	ShortTimeFormat = "01-02 15:04:05"
+	ShortTimeFormat = "2006-01-02 15:04:05"
```

Comment:
There's a small chance this might break any scripts that rely on this format, but meh.

Reply:
good point. I think this is worthwhile. we don't explicitly support an interface via stdout so whoever used it as an interface to their scripts will just have to update